src/screening/ScreeningFilters.jsx

```diff
@@ -1,50 +1,103 @@
 import React from "react";
 import {
   Dropdown,
-  FieldInput
 } from '@openmrs/react-components';
-import {
-  Glyphicon,
-  FormControl,
-} from 'react-bootstrap';
+import { FormControl } from 'react-bootstrap';
+import utils from "../utils";
 import './screeningFilters.css';
 
-let ScreeningFilters = props => (
-  <div className="queue-filters">
-    <div className="identifier-filter-container">
-      <div>ID search:</div>
-      <span className='identifier-filter'>
-        <Dropdown
-          dropDownStyle={{
-            border: '1px solid black'
-          }}
-          list={['mock-ID-1','mock-ID-2']} 
-          placeholder="mock-ID-0"
-        />
-        <span>-</span>
-        <FieldInput />
-        <span>-</span>
-        <Dropdown
-          dropDownStyle={{
-            border: '1px solid black'
-          }}
-          placeholder="mock-ID-0"
-        />
-      </span>
-    </div>
-    <div className="name-filter-container">
-      <div>Name/id search:</div>
-      <span className="name-filter">
-        <Glyphicon className="left-icon" glyph="search" />
-        <FormControl
-          autoFocus
-          placeholder="search by text"
-          type="text"
-        />           
-        <Glyphicon className="right-icon" glyph="remove-sign" />
-      </span>
-    </div>
-  </div>
-);
+class ScreeningFilters extends React.Component {
```

Comment:
@mogoodrich For the identifier filters, I am checking through all the identifiers on the cardList and using it to populate the filter dropdown, is that correct? I 

Also I checked all thorough the OpenMRS Wiki, I couldn't find the breakdown of a patient identifier.
I.e ABCD-1234-XZY,
I know ABCD, means something and XYZ means something, but I'm not sure what they are. So I had issue coming up with good variable names. I had to call them 
ABDC - `firstIdentifierSearchValue`
1234 - `secondIdentifierSearchValue`
XYZ - `thirdIdentifierSearchValue`
If you can help clarify what they mean, I'll update the code.

Reply:
@mseaton  @cioan  do you know what the appropriate names are for these components are, and what potential values are for the prefix and the suffix?